google_photos_takeout_helper/__main__.py

```diff
@@ -1,11 +1,10 @@
-import sys
-
+from tqdm import tqdm as _tqdm
 from loguru import logger
 
 logger.remove()  # removes the default console logger provided by Loguru.
 # I find it to be too noisy with details more appropriate for file logging.
 # INFO and messages of higher priority only shown on the console.
-logger.add(sys.stdout, format="{message}", level="INFO")
+logger.add(lambda msg: _tqdm.write(msg, end=""), format="{message}", level="INFO")
```

Comment:
Oh, it prints out logs without leaving dead bars after them... nice :sparkles: !

Reply:
Yes!